app/api/user_management.py

```diff
@@ -0,0 +1,56 @@
+from flask import Flask, jsonify, request
+from flask_restplus import Resource, Api, fields
+try:
+    from app.models import UserManagement
+except Exception:
+    from WeConnect.app.models import UserManagement
+
+
+user_data = []
+# login_info = []
+
+
+UserObj = UserManagement()
+
+class RegisterUsers(Resource):
+    
+    
+    
+    def post(self):
+        
+        user_data = request.json
+        email = user_data['email']
+        firstname = user_data['firstname']
+        secondname = user_data['secondname']
+        username = user_data['username']
```

Comment:
This code is unsafe and prone to bugs, accessing dictionary using the square bracket can throw exceptions when the key accessed is missing. Correct this by checking the key exists in the dictionary.